docker/1.11/Dockerfile.cpu

```diff
@@ -16,9 +16,11 @@ RUN \
     apt-get -y install --no-install-recommends nginx nginx-module-njs python3 python3-pip python3-setuptools && \
     apt-get clean
 
-# cython, falcon, gunicorn
+# cython, falcon, gunicorn, tensorflow-serving
 RUN \
-    pip3 install cython falcon gunicorn gevent requests
+    pip3 install cython falcon gunicorn gevent requests grpcio protobuf && \
+    pip3 install --no-cache-dir tensorflow && \
```

Comment:
`--no-cache-dir` is good. we should add to the other pip install commands too.

Reply:
Done.